pages/gallery.html

```diff
@@ -1,20 +1,62 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="es">
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="./../css/style.css" />
+    <link rel="stylesheet" type="text/css" href="./../css/gallery.css"/>
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
     <title>Galeria - Hostal las paisas</title>
+    
+
+
 </head>
 <body>
     <nav>
         <!-- nav bar section -->
     </nav>
-    <section>
+    <!-- gallery -->
+    <section class="container">
         <!-- gallery -->
+        <div class="info-gallery__text">    
+            <p>Nuestro hostal te ofrece todas las comodidades para que vivas la mejor experiencia. Observa todo lo que tenemos para ti en el majestuoso paraíso de Capurganá</p>
+        </div>
+            <div>
+                
```

Comment:
I believe this blank space can be removed.

Reply:
Ok, I already deleted it